ntbs-service/Models/Entities/Notification.cs

```diff
@@ -151,19 +151,29 @@ public bool ShouldBeClosed()
             var mostRecentTreatmentEvent = TreatmentEvents.GetMostRecentTreatmentEvent();
 
             return NotificationStatus == NotificationStatus.Notified 
-                   && mostRecentTreatmentEvent?.TreatmentOutcomeId != null 
-                   && mostRecentTreatmentEvent.TreatmentEventTypeIsOutcome
-                   && NotStillOnTreatmentAndOlderThanOneYear(mostRecentTreatmentEvent);
+                   && (EventIsOutcomeAndNotStillOnTreatmentAndOlderThanOneYear(mostRecentTreatmentEvent)
+                       || IsPostMortemAndOlderThanOneYear());
 
-            bool NotStillOnTreatmentAndOlderThanOneYear(TreatmentEvent treatmentEvent)
+            bool EventIsOutcomeAndNotStillOnTreatmentAndOlderThanOneYear(TreatmentEvent treatmentEvent)
```

Comment:
I wonder if this name could be simplified - would `EventIsEndingAndOlderThanOneYear` still be accurate?